src/main/java/core/basesyntax/JavaStreamApi.java

```diff
@@ -38,17 +46,24 @@ public Long elementCount(List<String> elements, String element) {
      * Найти элемент в коллекции равный `element` или кинуть ошибку NoSuchElementException</p>
      **/
     public String findElement(List<String> elements, String element) {
-        return null;
+        return elements.stream()
+                .filter(e -> e.contains(element))
+                .findFirst()
+                .orElseThrow(NoSuchElementException::new);
```

Comment:
look closely to orElseThrow() without parameters

Reply:
I see, i already throws such exception. Thanks!